README.md

```diff
@@ -59,14 +59,20 @@ about the files (name, mimetype).
 
 `--simple` creates a "human-readable" package; one folder per project or dataset is created and image files are placed according to where they came from in the OMERO server. Note that a package generated with this option is not guaranteed to work with `unpack`, though it often will.
 
-`--metadata` allows you to specify which transfer metadata will be saved in `transfer.xml` as possible MapAnnotation values to the images. Defaults to image ID, timestamp, software version, source hostname, md5, source username, source group. 
+`--metadata` allows you to specify which transfer metadata will be saved in `transfer.xml` as possible MapAnnotation values to the images. Defaults to image ID, timestamp, software version, source hostname, md5, source username, source group.
+
+`--metadata_only` creates the transfer.xml file but does not copy data
```

Comment:
Looking at this parallelism, I wonder if this is actually a `--binary` flag similar to the `--metadata` flag which takes options: `files (default), none,` and then in the future we could ask it to convert to OME-TIFF/OME-Zarr, for example. That doesn't help with the final argument handling or the exclusivity with simple/rocrate/etc.

Reply:
Here "binary" means all data that is not metadata?

Reply:
Generally, "big things that might be downloaded". File attachments might end up being a special case with my other examples (OME-TIFF/Zarr). This is certainly where `--server` was more straight-forward because it meant, *"I have access to all of the binaries and can do what I want with them"*

Reply:
at some point as well if we decide to be granular about which kinds of annotations are saved with the images as well. maybe 3 similar options covering what can be stored (`--metadata`, `--annotations`, `--binaries`) is the way to go?

Reply:
Maybe the confusion is lessened if this is ordered `metadata < annotation < full` `metadata` would then be a single file (`xml`, `tsv`, `jsonld`) and `annotation` everything but the  binaries. 

I don't see why you would want _only_ annotations or binaries